templating/base_render_options_test.go

```diff
@@ -3,7 +3,9 @@
 
 package templating
 
-import "testing"
+import (
```

Comment:
I've been wondering about this for a while?
What is the difference between the 2? I tried to look for it, but I only see a pattern of this being used when you import more than one file.

Reply:
There's no difference. It's an autocorrect pattern from `gofmt` starting in Go 1.11